migrations/1663773308937-Wallet.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Wallet1663773308937 implements MigrationInterface {
+  name = "Wallet1663773308937";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "wallet" ("id" SERIAL NOT NULL, "userId" integer NOT NULL, "walletAddress" character varying NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "UQ_35472b1fe48b6330cd349709564" UNIQUE ("userId"), CONSTRAINT "UK_wallet_id_walletAddress" UNIQUE ("id", "walletAddress"), CONSTRAINT "REL_35472b1fe48b6330cd34970956" UNIQUE ("userId"), CONSTRAINT "PK_bec464dd8d54c39c54fd32e2334" PRIMARY KEY ("id"))`,
```

Comment:
I don't know, maybe it's just me, but the Scraper API contains some DB models that are fundamental for blockchains:  `block`, `transaction`. In our case `wallet` is not any blockchain wallet, but a specific wallet linked by a user. What do you think if we rename this model to `UserWallet` (which translates to `user_wallet` table name)?

Reply:
Yes makes sense. I actually first named it like that 😅

Reply:
done